tasks/css.js

```diff
@@ -138,27 +138,25 @@ gulp.task('css:bundle', () => gulp.src(`${pathBuilder.scssSrcDir}/**/*.scss`)
         ])
     )
 
-    // output to docs assets folder
-    .pipe(
-        gulpif(config.docs.outputAssets,
-            gulp.dest(pathBuilder.docsCssDistDir)
-        )
-    )
-
     //add .min suffix to CSS files
     .pipe(rename({ suffix: ".min" }))
 
     // Output file-size
-    .pipe(
-        gulpif(config.misc.showFileSize,
-            tap(file => {
-                gutil.log(`❯❯ Minified CSS ${file.relative}`, filesizegzip(file.contents, true));
-            })
-        )
-    )
+    .pipe(gulpif(config.misc.showFileSize,
+        tap(file => {
+            gutil.log(`❯❯ Minified CSS ${file.relative}`, filesizegzip(file.contents, true));
+        })
+    ))
 
     // export sourcemaps (as a separate file)
-    .pipe(gulpif(config.isDev, sourcemaps.write(undefined, { sourceRoot: null })))
+    .pipe(gulpif(
+        config.isDev, sourcemaps.write(undefined, { sourceRoot: null })
+    ))
+
+    // output to docs assets folder
+    .pipe(gulpif(config.docs.outputAssets,
```

Comment:
Will outputting this later mean that the filename is different in the docs? (so `.min` will be added)

Just wondering as this might break some of the paths and they’ll just need to be updated too